### ISSUE TITLE
[FEATURE] - Launch script bash, sh, powershell, restart device

### ISSUE THREAD

**Issue by Apashh** [rustdesk/rustdesk-server-pro#58]
Hey,

It will be interesting to launch script or restart device from web console like https://github.com/immense/Remotely

Thanks !

**Comment by dinger1986** [comment]
I don't think this is a good idea for now. Maybe in the future adding scripts to be run would be good

**Comment by rustdesk** [comment]
You turn it to be a rmm.

**Comment by Apashh** [comment]
> You turn it to be a rmm.

what does it mean ?
